### PR TITLE
Add initial code for breakout game

### PR DESCRIPTION
**Pull request by davidthomas22**

### CODE REVIEW

File: moj-node/assets/javascript/games/breakout/index.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
+var gameCanvas = document.getElementById("gameCanvas");
```
Comment (psoleckimoj) [inline moj-node/assets/javascript/games/breakout/index.js]: Just needs the tabbing sorting for formatting :)

Comment (psoleckimoj) [reply moj-node/assets/javascript/games/breakout/index.js]: This initial block may need to be in a function called by onLoad? It's possible that the code runs before the HTML has been generated and thus gameCanvas won't exist?

Comment (psoleckimoj) [reply moj-node/assets/javascript/games/breakout/index.js]: First example here... https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event

Comment (davidthomas22) [reply moj-node/assets/javascript/games/breakout/index.js]: function applied to window.onload

**Pull request merged**